swap_meet/vendor.py

```diff
@@ -1,2 +1,55 @@
+from .item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = inventory if inventory is not None else []
+
+    def add(self, add_item):
+        self.inventory.append(add_item)
+        return add_item
+    
+    def remove(self, remove_item):
+        if remove_item not in self.inventory:
+            return False
+        
+        self.inventory.remove(remove_item)
+        return remove_item
+    
+    def get_by_category(self, item_category):
+        items_list = [item for item in self.inventory if item.category == item_category]
+
+        return items_list
```

Comment:
Nice list comprehension here! You can also directly return the list without saving it to a variable first if you'd like:

```python 
return [item for item in self.inventory if item.category == item_category]
```